cluster-install/install.sh

```diff
@@ -2315,14 +2355,19 @@ step_monitor_deployment_progress() {
     if trident_will_be_installed_or_modified; then
         if is_dry_run; then
             logdebug "skip monitoring trident components because it's a dry run"
-        elif ! wait_for_deployment_running "trident-operator" "$trident_ns" "1m"; then
+        elif ! wait_for_deployment_running "trident-operator" "$trident_ns" "3"; then
             add_problem "trident operator: failed" "The Trident Operator failed to deploy"
-        elif ! wait_for_deployment_running "trident-controller" "$trident_ns" "10m"; then
-            add_problem "trident controller: failed" "Trident failed to deploy"
+        elif ! wait_for_cr_state "torc/$_EXISTING_TORC_NAME" ".status.status" "Installed" "$trident_ns" "12"; then
```

Comment:
Small change to tracking Trident deployment progress; using **torc** status now instead of the **trident-controller** deployment, the former of which I assume is more reliable since the operator sets it.